internal/integration_test/vs/runtime.go

```diff
@@ -86,6 +89,11 @@ func (r *wazeroRuntime) Compile(ctx context.Context, cfg *RuntimeConfig) (err er
 			ExportFunction("log", r.log).Compile(ctx, wazero.NewCompileConfig()); err != nil {
 			return err
 		}
+	} else if cfg.EnvFReturnValue != 0 {
+		if r.env, err = r.runtime.NewModuleBuilder("env").
+			ExportFunction("f", func(uint64) uint64 { return cfg.EnvFReturnValue }).Compile(ctx, wazero.NewCompileConfig()); err != nil {
```

Comment:
It would be good to get numbers for various number of arguments, and especially with `context.Context` and `api.Module` included

Reply:
agreed. maybe we can to keep dimensions under wraps, compare one more with a memory write (offset/len)? This would use the context/mod arg and is extremely typical, probably more typical than pure function.

Reply:
oh ok

Reply:
turns out the same result with this https://github.com/tetratelabs/wazero/pull/756/commits/c0d7682a9eb8b0fe7296233a7b826fbbf1fbfad3 

I just default it here, and will do the comparison (with/without api.Module and context) in wasm vs host bench in another PR